data/172/936/307/3/1729363073.geojson

```diff
@@ -0,0 +1,47 @@
+{
+  "id": 1729363073,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.469982,
+    "geom:area_square_m":4476094505.940491,
+    "geom:bbox":"173.281863,-39.85994,174.656439,-39.285251",
+    "geom:latitude":-39.625794,
+    "geom:longitude":173.749268,
+    "iso:country":"",
+    "lbl:latitude":-39.571203,
+    "lbl:longitude":173.555516,
+    "mps:latitude":-39.571203,
+    "mps:longitude":173.555516,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "name:eng_x_preferred":[
+        "South Taranaki Bight"
+    ],
+    "src:geom":"nz-linz",
+    "src:lbl_centroid":"mapshaper",
+    "wof:belongsto":[],
+    "wof:breaches":[],
+    "wof:country":"",
+    "wof:created":1602807776,
+    "wof:geomhash":"066953f59afb7fe38d40bce31f45226a",
+    "wof:hierarchy":[],
+    "wof:id":1729363073,
+    "wof:lastmodified":1602807776,
+    "wof:name":"South Taranaki Bight",
+    "wof:parent_id":-1,
```

Comment:
Are these supposed to be parented by the 7 Seas continent or by NZ country or?

Reply:
The parent_id/hierarchy for these records hasn't really been flushed out.. an example of the current record for Great Australia Bight has an empty hierarchy and parent_id of `-1`: https://spelunker.whosonfirst.org/id/404528847/

Since these are within a revgeo boundary/in NZ waters, I think it would be fair to include country and continent in their hierarchies..